src/app/page.tsx

```diff
@@ -1,5 +1,30 @@
+import CountryCard from '@/components/CountryCard';
 import styles from './page.module.scss';
+import Link from 'next/link';
+import getDataForHomepage from './fetchUtils/getDataForHomepage';
+import { TCountry } from '@/app/types/countryTypes';
 
-export default function Home() {
-  return <main className={styles.main}></main>;
+export default async function Home() {
+  const data = await getDataForHomepage();
+  const fetchedList = await data;
```

Comment:
we can refactor this instead of the current approach
```
const data = await getDataForHomepage();
const fetchedList = await data;
```

you can do this
```
const fetchedList = await getDataForHomepage();
```

Reply:
That does look better, refactored.